rclpy/test/test_client.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
```

Comment:
What are the non-ideal cases? How long do they take? I upped it by 30 seconds (+4ish seconds per new test) 72e8ea94f86c807f393d68430c0bb8e8d98fd121.

Reply:
The worst case scenario really depends on the nature of the test, e.g. if a test has a wait_for_service(5s) worst case it will time out and take ~5seconds. For reference our service communication tests have a timeout of 30s. On my machine, this takes between 90 and 100s so I guess 120 is fine. We often revisit test timeout later on given that not all nodes have the same behavior. A good way to estimate the duration would be to run jobs on all platforms with this single test repeated multiple times